src/lib/ProductExporter.ts

```diff
@@ -71,17 +79,28 @@ export class ProductExporter {
         Util.showSpinner('products export');
 
         let productDefinitions = await Queries.queryProduct(conn, productList);
+        Util.isBulkApi(productDefinitions) ? productDefinitions = await Queries.bulkQueryProduct(conn, productList) : null;
```

Comment:
@adrianlukaszek1992 co ta konstrukcja ma na celu?

- w linii 81 do zmiennej productDefinitions przypisujesz rezultat query
- a potem w linii 82 jeśli isBulkAPI to ponownie strzelasz query i przypisujesz rezultat do tej zmiennej ?

O co tutaj chodzi?